design/sketch.webidl

```diff
@@ -452,6 +459,8 @@ dictionary WebGPURenderPipelineDescriptor : WebGPUPipelineDescriptorBase {
     WebGPUDepthStencilState depthStencilState;
     WebGPUInputState inputState;
     WebGPUAttachmentsState attachmentsState;
+    // Number of MSAA samples for
+    u32 samples;
```

Comment:
so this `u32 samples` is compared to the `u32 samples` below? I'm a little confused.